pages/authentication.page.ts

```diff
@@ -0,0 +1,56 @@
+require('selenium-webdriver/chrome');
+const {By, Key, until } = require('selenium-webdriver');
+export class AuthentiPage{
+    private driver;
+    private accountEmail;
+    private loginPassword;
+    private loginEmail;
+    private firstName;
+    private lastName; 
+    private passwd; 
+    private address; 
+    private city; 
+    private state; 
+    private postcode
+    private phone;
+
+    constructor(driver){
+        this.driver = driver;
+    }
+
+//#region methods
+    public async submitAccountEmail(email: string){
+        this.accountEmail = await this.driver.findElement(By.id('email_create'))
+        this.accountEmail.sendKeys(email);
+        await this.driver.findElement(By.id('SubmitCreate')).click();
+        await this.driver.sleep(5000);
+    }
+
+    public async submitSignInData(email: string, passwd : string){
+        this.loginEmail = await this.driver.findElement(By.id('email')).sendKeys(email);
+        this.loginPassword = await this.driver.findElement(By.id('passwd')).sendKeys(passwd);
+        await this.driver.findElement(By.id('SubmitLogin')).click();
+        await this.driver.sleep(500);
+    }
+
+    public async submitSignUpData(firstName: string, lastName : string,passwd: string,address: string,city: string, state: string, postcode: string,phone: string){
+        this.firstName = await this.driver.findElement(By.id('customer_firstname')).sendKeys(firstName);
+        this.lastName = await this.driver.findElement(By.id('customer_lastname')).sendKeys(lastName);
```

Comment:
already suggest that locators should not be in the methods; it is better approach that there are variables for locators that will be used